v2/chapter_toc_frame.go

```diff
@@ -0,0 +1,141 @@
+package id3v2
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"io"
+)
+
+const (
+	maskOrdered  = byte(1 << 0)
+	maskToplevel = byte(1 << 1)
+)
+
+var ErrUnexpectedId = errors.New("unexpected ID")
+
+type ChapterTocFrame struct {
+	ElementID string
+	// This frame is the root of the Table of Contents tree and is not a child of any other "CTOC" frame.
+	TopLevel bool
+	// This provides a hint as to whether the elements should be played as a continuous ordered sequence or played individually.
+	Ordered     bool
+	ChapterIds  []string
+	Description *TextFrame
```

Comment:
I think it should be named as `DescriptiveData` like in spec:
```suggestion
	DescriptiveData *TextFrame
```

Reply:
I think my implementation is wrong. After reading the spec again: TIT2 and TIT3 can be added, so it's a collection of text frames rather than one single text frame.